### PR TITLE
gpperfmon overview improvements

### PR DESCRIPTION
**Pull request by litzell**
Clarifies how alert logs are captured and rotated, adds gpmmon/gpsmon process overview.

### CODE REVIEW

File: gpdb-doc/dita/ref_guide/gpperfmon/dbref.xml [hunk 1]
```diff
@@ -62,5 +66,128 @@
...
+        Enterprise Linux, for example, it is 4096 bytes. If many Greenplum Database sessions
+        generate error messages at the same time, the log file can grow significantly before its
+        size is checked and log rotation is triggered.</p>
+    </section>
+    <section id="rotation">
+      <title>gpperfmon Data Collection Process</title>
+      <p>When Greenplum Database starts up with gpperfmon support enabled, it forks a
+          <codeph>gpmmon</codeph> agent process. <codeph>gpmmon</codeph> then starts a
+          <codeph>gpsmon</codeph> agent process on the master host and every segment host in the
+        Greenplum cluster. The Greenplum postmaster process monitors the <codeph>gpmmon</codeph>
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/gpperfmon/dbref.xml]: Greenplum -> Greenplum Database

File: gpdb-doc/dita/ref_guide/gpperfmon/dbref.xml [hunk 2]
```diff
@@ -62,5 +66,128 @@
...
+        process and restarts it if needed and the <codeph>gpmmon</codeph> process monitors and
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/gpperfmon/dbref.xml]: add comma:  if needed,

File: gpdb-doc/dita/utility_guide/admin_utilities/gpperfmon_install.xml [hunk 3]
```diff
@@ -183,8 +182,8 @@ host       all           gpmon  ::1/128        <b>password</b></codeblock></p>
         </strow>
         <strow>
           <stentry>partition_age</stentry>
-          <stentry>The number of months that gperfmon statistics data will be retained. The default
-            it is 0, which means we won’t drop any data.</stentry>
+          <stentry>The number of months that gpperfmon statistics data will be retained. The default
+            it is 0, which means that partitions are never dropped automatically.</stentry>
```
Comment (dyozie) [inline gpdb-doc/dita/utility_guide/admin_utilities/gpperfmon_install.xml]: "it is" -> "value is"

**Comment by litzell** [comment]
These are links to these documents rendered for review:

- http://docs-gpdb-review-staging.cfapps.io/500Beta/ref_guide/gpperfmon/dbref.html
- http://docs-gpdb-review-staging.cfapps.io/500Beta/utility_guide/admin_utilities/gpperfmon_install.html

File: gpdb-doc/dita/utility_guide/admin_utilities/gpperfmon_install.xml [hunk 4]
```diff
@@ -195,6 +194,12 @@ host       all           gpmon  ::1/128        <b>password</b></codeblock></p>
               frequently, choose a lower value.</p></stentry>
         </strow>
         <strow>
+          <stentry>harvest_interval</stentry>
```
Comment (cjcjameson) [inline gpdb-doc/dita/utility_guide/admin_utilities/gpperfmon_install.xml]: Sweet, you had mentioned needing this. Good to put it in.

**Comment by skahler-pivotal** [comment]
LGTM

**Pull request merged**